	Initialized static const class members may need to be defined
	in the enclosing scope.

diff -ur engine-2.4.orig/test/RegressionTest/RegressionTest.cpp engine-2.4/test/RegressionTest/RegressionTest.cpp
--- engine-2.4.orig/test/RegressionTest/RegressionTest.cpp	2010-06-09 23:26:29.000000000 +0200
+++ engine-2.4/test/RegressionTest/RegressionTest.cpp	2012-03-17 00:34:34.000000000 +0100
@@ -34,6 +34,7 @@
 
 SegmentMemoryUsage g_smu;
 FontMemoryUsage g_fmu;
+const int TestCase::kAbsent;
 
 std::string g_fontPath = ".";
 #ifdef _WIN32
diff -ur engine-2.4.orig/test/RegressionTest/TestCase.cpp engine-2.4/test/RegressionTest/TestCase.cpp
--- engine-2.4.orig/test/RegressionTest/TestCase.cpp	2011-02-28 19:20:51.000000000 +0100
+++ engine-2.4/test/RegressionTest/TestCase.cpp	2012-03-17 00:34:34.000000000 +0100
@@ -2451,6 +2451,7 @@
 		m_attGlyphs[i] = stuff[i];
 }
 
+const int ClickTest::fieldCnt;
 void TestCase::SetClickTests(int clickTestCount, int * clickStuff)
 {
 	const int fc = ClickTest::fieldCnt;
